src/Native/jitinterface/jitwrapper.cpp

```diff
@@ -78,3 +89,12 @@ DLL_EXPORT int JitCompileMethod(
 
     return 1;
 }
+
+DLL_EXPORT unsigned GetMaxIntrinsicSIMDVectorLength(
+    Jit * pJit,
+    CORJIT_FLAGS flags)
+{
+    // BUGBUG: the signature on the Jit side is actually "pass by value", but for reasons that escape me,
```

Comment:
This may have to do with different calling convention for Plain-Old-Datatype structs vs. C++ structs. Does it change if you add constructor to CORJIT_FLAGS ?